tests/components/sidebar/sidebar.test.jsx

```diff
@@ -1,478 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Did you mean to remove this whole file?

Reply:
Yes. Every test in this file is broken. Ticket created to fix.

Reply:
Let me know the ticket, so I could take a look at this.  Thanks